subprojects/core/src/main/java/org/gradle/api/internal/file/DefaultProjectLayout.java

```diff
@@ -104,6 +104,7 @@ public FileCollection files(Object... paths) {
     }
 
     @Override
+    @Deprecated
```

Comment:
Why does this need repeating here?

Reply:
Otherwise you get a deprecation warning because you implement a deprecated interface method (alternatively we could do a suppress warning).

Reply:
I don't get that warning with the default IntelliJ settings in 2020.1.1.

Reply:
I see your comments below. I think it's fine to add these. Would be good to align IntelliJ warnings, too.